k8s_api_client/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+resources:
+- resources/serviceaccount.yaml
+- resources/secret.yaml
+- resources/deployment.yaml
+- resources/ingress.yaml
+- resources/rbac.yaml
+- resources/service.yaml
+configMapGenerator:
+- name: opa-kubernetes-api-client
+  files:
+  - policy/api_client.rego
```

Comment:
💡 TIL. I'll use something that in the future. 😃

Reply:
Yeah that is very useful indeed - the generator applies a randomized suffix to the name as well so you can change the policy file, re-run `kubectl apply -k .` and have the new policy deployed.